### DISCUSSION TITLE
how do I add the "hide artist" option?

### DISCUSSION THREAD

**Discussion by raflz** [ampache/ampache#3230] [Q&A]
Hi. 
I can hide the song.
How do I add the "hide artist", "hide album" option?
Ampache 5.3.1

**Comment by mitchray** [comment]
You could enable ```rating_browse_filter``` and ```rating_browse_minimum_stars``` in the config/ampache.cfg.php file

Just remove the ```;``` on each to enable it, then any artists and albums you rate as 1 star will be hidden

**Reply by raflz** [reply]
will they be hidden for everyone except the administrator?

**Reply by raflz** [reply]
it didn't help. albums and artists are visible to another user

**Reply by mitchray** [reply]
Catalogs can be either public or private to a user, that might be better depending on your setup

Can you describe your setup and goals with hiding things?

**Reply by raflz** [reply]
all default settings

**Reply by mitchray** [reply]
I meant what is the reason for wanting to hide things, so we can give better advice

**Reply by raflz** [reply]
I need to hide some songs, albums, artists from children

**Comment by mitchray** [comment]
You may want to follow this issue which seems identical to your situation https://github.com/ampache/ampache/issues/2993

Presently the only solution is to make a separate child-only catalog, with the children sharing an account that can only see that catalog

**Reply by raflz** [reply]
thx

**Comment by troycarpenter** [comment]
Be aware that in my testing, that's not necessarily sufficient.  I have some comments...

First, you can't limit the kids account to only that catalog.  A private catalog means that user is the only one that can see it, not that it's the only catalog the user can see.  Any catalog that is public can still be seen.

Secondly, a private catalog can currently only be private to a single user, not a set of users unless they share an account.

Finally, when I restricted a catalog to only me, the other users could still see the content was present on the server (album covers and all), but only when you drilled down to the album itself did it show zero tracks to other users and they couldn't play content from that catalog.  This may have been a side effect of artists in both catalogs, but I think the search may have still revealed restricted items.

**Comment by troycarpenter** [comment]
I have started preliminary work on this feature.  I have a discussion thread on #3244
